client/src/components/Navbar.js

```diff
@@ -1,110 +1,112 @@
-import React,{useContext, useEffect} from 'react';
-import {Toolbar, AppBar, Box, Typography, Link, IconButton, Button} from '@material-ui/core';
+import React, { useContext, useEffect } from 'react';
+import PhotoUpload from './PhotoUpload.js';
+import { Toolbar, AppBar, Box, Typography, Link, IconButton, Button } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import { Link as RouterLink } from 'react-router-dom';
 import LocalMallIcon from '@material-ui/icons/LocalMall';
 import AuthContext from '../state_management/AuthContext';
-import {fetchShLists} from '../state_management/actionCreators/shoppingListsActs';
+import { fetchShLists } from '../state_management/actionCreators/shoppingListsActs';
 import ShListsContext from '../state_management/ShListsContext';
+import FindNewFriendsModal from '../components/FindNewFriendsModal.js';
 
 const useStyles = makeStyles((theme) => ({
-    appBar: {
-      borderBottom: `1px solid ${theme.palette.divider}`,
-    },
-    toolbar: {
-      flexWrap: 'wrap',
-    },
-    toolbarTitle: {
-      flexGrow: 1,
-    },
-    link: {
-      margin: theme.spacing(1, 1.5),
-    },
-    margin:{
-        margin: theme.spacing(1),
-    }
-  }));
+	appBar: {
+		borderBottom: `1px solid ${theme.palette.divider}`
+	},
+	toolbar: {
+		flexWrap: 'wrap'
+	},
+	toolbarTitle: {
+		flexGrow: 1
+	},
+	link: {
+		margin: theme.spacing(1, 1.5)
+	},
+	margin: {
+		margin: theme.spacing(1)
+	}
+}));
 
 let needsFetchingLists = true;
 let needsCleanList = false;
 
-const Navbar = ()=>{
-    const classes = useStyles();
-    const authContext = useContext(AuthContext);
-    const shListsContext = useContext(ShListsContext);
-    
+const Navbar = () => {
+	const classes = useStyles();
+	const authContext = useContext(AuthContext);
+	const shListsContext = useContext(ShListsContext);
 
-    useEffect(() => {
-        if(authContext.isAuthenticated && needsFetchingLists){ //pretects, because private route is uncommented for developing
-            fetchShLists(shListsContext.dispatchShLists,shListsContext.handleShListsFailure);
-            needsFetchingLists = false;
-            console.log("Navbar/useEffet is fetching");
-        }
-        if(needsCleanList){
-            shListsContext.handleShListsFailure({response:null});
-            needsCleanList = false;
-        }
-    });
+	useEffect(() => {
+		if (authContext.isAuthenticated && needsFetchingLists) {
```

Comment:
We might have merge conflicts here as Irene also updated same file with socket.io